src/shared/commons/collections-helpers.ts

```diff
@@ -0,0 +1,14 @@
+export class CollectionsHelpers {
+  constructor() {
+  }
+
+  filterElement(el) {
+    return el != null;
+  }
+
+  flatAnArray(array) {
```

Comment:
I assume we have already checked whether javasctipt provide a fatten method already right?

Reply:
Array.prototype.flat is not supported by in our Karma test settings. We need to increase ECMAScript.  It works in the browser but it fails im the yarn test .

Reply:
fair enough 👍